src/chrome/content/scaffold/templates/scrapeEM.js

```diff
@@ -12,7 +12,7 @@ function scrape(doc, url) {
 
 	translator.getTranslatorObject(function(trans) {
 		trans.itemType = "newspaperArticle";
-		//TODO adjust if needed:
+		//You can map additional meta tags here
```

Comment:
Keep the TODO, since the comment doesn't make sense once out of the context of the initial translator development

Reply:
Done